_maps/RandomRuins/SpaceRuins/dangerous_research.dmm

```diff
@@ -3201,9 +3201,6 @@
 /area/ruin/space/has_grav/dangerous_research/lab)
 "Pc" = (
 /obj/effect/decal/cleanable/dirt/dust,
```

Comment:
does this decal even work on closed turfs? it doesn't seem like it should

Reply:
Sure, will do.